ApollosProject/apollos-apps
# feat: Auto follow

## DESCRIPTION
Automatically accept follow requests to suggested people

### What does this PR do, or why is it needed?
When creating a new follow record, if the person being followed is one of the suggested people, the record is created with the state already set to `ACCEPTED`.

### How do I test this PR?
Run the templates app and request to follow one of the suggested users on the connect tab. Check the database, and the status should be `ACCEPTED`.
